### PR TITLE
Readonly controlboard

### PR DESCRIPTION
**Pull request by francesco-romano**
and remote remapper.

### CODE REVIEW

**Comment by francesco-romano** [comment]
@traversaro: @claudia-lat confirmed that it seems to work.
If you have some comments, otherwise I will rebase and merge.

File: yarp/ReadOnlyControlBoardRemapper/ReadOnlyRemoteControlBoardRemapper.h [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+/**
+ *  @ingroup dev_impl_wrapper
+ *
+ * \section RemoteControlBoardRemapper
+ * A device that takes a list of axes from multiple controlboards, a list
+ * of remote controlboards in which this axes are located, that is opening
+ * all the remote controlboards but is exposing them
+ *
+ *
+ *  Parameters required by this device are:
```
Comment (traversaro) [inline yarp/ReadOnlyControlBoardRemapper/ReadOnlyRemoteControlBoardRemapper.h]: I think Markdown tables need a space before and after to be rendered, but I am not sure.

File: yarp/ReadOnlyControlBoardRemapper/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,28 @@
+YARP_PREPARE_PLUGIN(readonlyremotecontrolboardremapper TYPE yarp::dev::ReadOnlyRemoteControlBoardRemapper
+                                                       INCLUDE ReadOnlyRemoteControlBoardRemapper.h
+                                                       CATEGORY device)
+
+if(ENABLE_readonlyremotecontrolboardremapper)
+
+  set(PLUGIN_SOURCES "${CMAKE_CURRENT_SOURCE_DIR}/ReadOnlyControlBoardRemapper.cpp"
```
Comment (francesco-romano) [inline yarp/ReadOnlyControlBoardRemapper/CMakeLists.txt]: yes.. I think so.. I was too explicit? Do you think I should remove it?

File: yarp/ReadOnlyControlBoardRemapper/ReadOnlyRemoteControlBoardRemapper.h (hunk 1 above)
Comment (francesco-romano) [inline yarp/ReadOnlyControlBoardRemapper/ReadOnlyRemoteControlBoardRemapper.h]: It has been copy-pasted from the original source. Still to be modified with the new parameters

File: yarp/ReadOnlyControlBoardRemapper/CMakeLists.txt (hunk 2 above)
Comment (traversaro) [inline yarp/ReadOnlyControlBoardRemapper/CMakeLists.txt]: No problem in keeping it, just wondering if I was missing something.

File: yarp/ReadOnlyControlBoardRemapper/ReadOnlyRemoteControlBoardRemapper.h (hunk 1 above)
Comment (traversaro) [inline yarp/ReadOnlyControlBoardRemapper/ReadOnlyRemoteControlBoardRemapper.h]: Ok, then it works fine http://wiki.icub.org/codyco/dox/html/classyarp_1_1dev_1_1RemoteControlBoard.html .

**Pull request merged**